recipes/rdata/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rdata" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rdata-{{ version }}.tar.gz
+  sha256: 007c80d850f984de9103771531ab9c8c46676ea290249ff20ba48ef11168257e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - numpy
+    - pandas
+    - python >=3.6,<4.0
+    - setuptools
```

Comment:
Does this need setuptools at runtime?

Reply:
Humm, no, in fact it doesn't. I guess this is a mistake for my part in the setup.py requirements. It does not make sense to require setuptools if the setup.py script is importing setuptools already, after all!

So, I suppose that I should create a pyproject.toml file and move the setuptools requirement to it. As far as I know Conda does not use setup.py or pyproject.toml at all (is that right?), so it could then be removed fron the recipe. Should I do this before the recipe is accepted, or can you accept this and I then change the recipe in the feedstock?

Reply:
We like to have things right before we create a feedstock and package

Reply:
Ok, it should be right now.